distribution/docs/src/main/resources/_contents/_core-concepts/content-intro-contents.adoc

```diff
@@ -1,6 +1,8 @@
 
-placeholder text.
-////
-TODO: write this section.
-See https://codice.atlassian.net/browse/DDF-2471
-////
+The Catalog Framework can interface with Storage providers to provide storage of resources to specific types of storage, e.g., file system, relational database, XML database.
+A default file system provider is provided by default.
+Storage plugins provide pluggable functionality that can be executed either immediately before or immediately after content has been stored or updated.
+Storage providers act as a proxy between the Catalog Framework and the mechanism storing the content, e.g., file system, relational database.
```

Comment:
Is it worth linking to the `storage providers` section here? Not sure what our threshold or guidance is on links.